vuecomponents/tool/src/main/frontend/src/components/dynamic-criterion.vue

```diff
@@ -0,0 +1,164 @@
+<template>
+  <div class="crit-background" aria-describedby="title">
+    <div class="row align-items-center">
+      <div class="col-md-2">
+        <button @click="toggleColor" :class="[btnClass, 'select-btn']" :disabled="saving || editing" :title="i18n.select_criterion">{{idpos+1}}</button>
+        <input class="points"
+          v-if="saving && editing"
+          v-model="criterion.pointsVal"
+          @change="inputChange"
+          :aria-label="i18n.criterion_points"
+          type="Number" />
+        <span class="points" v-if="!saving || !editing">{{Number(criterion.pointsVal).toFixed(2)}}</span>
+      </div>
+      <div class="col-md-9">
+        <input class="descr"
+          v-if="saving && editing"
+          @change="inputChange"
+          :aria-label="i18n.criterion_description"
+          maxlength="255"
+          v-model="criterion.description" />
+        <span v-if="!saving || !editing">{{criterion.description}}</span>
+      </div>
+      <div class="col-md-1">
+        <button class="no-border" v-if="saving && editing" @click="saveChanges" :title="i18n.confirm_criterion">
+          <font-awesome-icon :style="{color: 'green'}" icon="fa-solid fa-check" />
+        </button>
+        <button class="no-border" v-if="saving && !editing" @click="editing=true" :title="i18n.edit_criterion">
+          <font-awesome-icon :style="" icon="fa-solid fa-pencil" />
+        </button>
+        <button class="no-border" v-if="saving" @click="deleteRow" :title="i18n.remove_criterion">
+          <font-awesome-icon :style="{color: 'red'}" icon="fa-solid fa-xmark" />
+        </button>
+      </div>
+    </div>
+  </div>
+</template>
+
+<style>
+.crit-background {
+  border: 1px solid var(--sakai-border-color);
+  border-radius: 6px;
+  background-color: var(--sakai-background-color-2);
+  max-width: 1080px;
+}
+
+.descr {
+    width: 100%;
+}
+.points {
+    max-width: 70px;
+    margin-left: 25px;
+}
+
+.svg-inline--fa {
+  width: 1em;
+}
+
+.select-btn {
+  padding:5px 10px;
+  color: #fff;
+  border: none; 
+  margin: 5px;
+}
+.select-btn:disabled {
+  background-color: black !important; 
+}
+.no-border {
+  border: none;
+}
+
+.red-color {
```

Comment:
Could this classes be usefull here? https://getbootstrap.com/docs/5.3/utilities/background/#background-color

Reply:
Yes, thanks, this is probably a leftover of when I started with no bootstrap.